saved-sites/saved-sites-store/src/main/java/com/duckduckgo/savedsites/store/SavedSitesEntitiesDao.kt

```diff
@@ -66,13 +66,15 @@ interface SavedSitesEntitiesDao {
     )
     fun allEntitiesInFolderSync(folderId: String): List<Entity>
 
-    @Query("update entities set deleted = 1, lastModified = :lastModified where entityId = :id")
+    @Query("update entities set title = '', url = '', deleted = 1, lastModified = :lastModified where entityId = :id")
     fun delete(
         id: String,
         lastModified: String = DatabaseDateFormatter.iso8601(),
     )
 
-    @Query("update entities set deleted = 1, lastModified = :lastModified where entityId != :bookmarksRoot AND entityId != :favoritesRoot")
+    @Query(
```

Comment:
Curious, why are not just deleting the entry in the db?

Reply:
This is called from the DevSettings screen, so I don’t see why we wouldn’t. Thanks for spotting that @aitorvs .